lib/routes/routes.dart

```diff
@@ -23,28 +24,43 @@ const String Folder3Path = '/folder/:fid/folder2/:f2id/folder3/:f3id';
 const String Player3 = '/folder/:fid/folder2/:f2id/folder3/:f3id/session/:sid';
 const String UrlPath = '/url';
 const String CollectionPath = '/app';
-const String webviewPath = '/webview';
-const String backgroundSounds = '/backgroundsounds';
+const String WebviewPath = '/webview';
+const String BackgroundSoundsPath = '/backgroundsounds';
```

Comment:
Hmm effective Dart says to prefer lowerCamelCase for consts
https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

What do you think? We would need to update all the others too I suppose, like the ones in Tracking

Reply:
haha, I saw some of them were starting with the camel case so that's I changed it otherwise I always prefer to use the lowerCamelCase for consts.